pkg/translator/aggregator_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright © 2022 Kubernetes Authors
+Copyright © 2023 Kubernetes Authors
```

Comment:
Generally we don't bump the year if we're just modifying earlier content

Reply:
Reverted